src/shim/runtime/v2/rune/config/config.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import (
+	"encoding/json"
+	"os"
+
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+// LoadSpec loads the specification from the provided path.
+func LoadSpec(cPath string) (spec *specs.Spec, err error) {
+	cf, err := os.Open(cPath)
+	if err != nil {
+		return nil, err
+	}
+	defer cf.Close()
+	if err := json.NewDecoder(cf).Decode(&spec); err != nil {
+		return nil, err
+	}
+	return spec, nil
+}
+
+func SaveSpec(cPath string, spec *specs.Spec) error {
+	data, err := json.Marshal(spec)
+	if err != nil {
+		return err
+	}
+	return os.WriteFile(cPath, data, 0644)
```

Comment:
Could these perms be `0640` or even `0600`? I'd also consider making it a global `const` to make it as visible as possible (for auditing, and potential re-use).

Reply:
This file is the config.json of the container, 0644 should be better.